src/main/java/com/synopsys/integration/blackduck/service/bucket/BlackDuckBucketFillTask.java

```diff
@@ -38,16 +41,19 @@ public BlackDuckBucketFillTask(final BlackDuckService blackDuckService, final Bl
     }
 
     @Override
-    public void run() {
+    public Optional<? extends BlackDuckResponse> call() {
```

Comment:
as this represents a breaking api change, we'd also want to update the build.gradle to reflect the new semantic version